src/box/memtx_tx.c

```diff
@@ -1234,8 +1234,6 @@ memtx_tx_story_unlink_both_on_rollback(struct memtx_story *story, uint32_t idx)
 			memtx_tx_track_read_story_slow(tracker->reader,
```

Comment:
By the way, the function has a bad comment:
*As opposed to `memtx_tx_story_unlink_both_on_rollback`*

Reply:
Fixed the typo, thanks!